manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -33,6 +33,10 @@ defaults:
 
     env: {}
 
+    taint:
+      enabled: false
+      namespace: ""
+
```

Comment:
Does change this require update istio install using helm or istioctl?

e.g. do we need this be a runtime config change?

is namespace a list of namespaces (app runs) or istio-cni's namespace?

Reply:
I saw it is for istio-cni namespace, would be good to clarify it or update the field to be clear.

Reply:
Yes, i agree that the namespace is a bit confusing there. I'll try and clarify.
This is meant for ambient, but can be useful for sidecar users as well now that i think about it

Reply:
@ilrudie maybe you could clarify this in the comments? `namespace: ""` is only useful when taint is enabled?  I assume it is a list of namespaces?

Reply:
Thanks Lin, I had not noticed this comment was still pending. I have added some documentation around the new fields in the latest commit.